UrlTest.php

```diff
@@ -196,9 +196,13 @@ public function isValid(): bool
                     $this->valid = false;
                 }
             }
-            $responseHeaderNames = array_keys($this->getResponse()->getHeaders());
+            $responseHeaderNames = array_map(function ($responseHeader) {
+                return strtolower($responseHeader);
+                }, array_keys($this->getResponse()->getHeaders())
+            );
```

Comment:
$responseHeaderNames = array_map(function ($responseHeader) {
    return strtolower($responseHeader);
}, array_keys($this->getResponse()->getHeaders()));